pytests/repo/setup-repo.sh

```diff
@@ -123,6 +128,15 @@ gpgcheck=0
 enabled=1
 EOF
 
+cat << EOF > ${TEST_REPO_DIR}/yum.repos.d/photon-test-src.repo
```

Comment:
one question:
what happens if my binary rpm and source rpm are in same repo?

Reply:
That should be totally fine.